library/blob.c

```diff
@@ -2028,11 +2025,32 @@ static int eblob_write_prepare_disk(struct eblob_backend *b, struct eblob_key *k
 			copy, copy_offset, old);
 
 err_out_exit:
+	return err;
+}
+
+
+/**
+ * eblob_write_prepare_disk() - allocates space for new record
+ * It allocates new bases, commits headers and manages overwrites/appends.
+ */
+static int eblob_write_prepare_disk(struct eblob_backend *b, struct eblob_key *key,
+				    struct eblob_write_control *wc, uint64_t prepare_disk_size,
+				    enum eblob_copy_flavour copy, uint64_t copy_offset, struct eblob_ram_control *old,
+				    size_t defrag_generation)
+{
+	ssize_t err = 0;
+	eblob_log(b->cfg.log, EBLOB_LOG_NOTICE,
+		  "blob: %s: eblob_write_prepare_disk: start: "
+		  "size: %" PRIu64 ", offset: %" PRIu64 ", prepare: %" PRIu64 "\n",
+		eblob_dump_id(key->id), wc->size, wc->offset, prepare_disk_size);
```

Comment:
Some is wrong with indentation, column is violated.

Reply:
@sabramkin check it now, please.